### DISCUSSION TITLE
期待windows版本

### DISCUSSION THREAD

**Discussion by whiteS18** [linuxdeepin/developer-center#8365] [Features Request & Ideas | 特性请求 & 头脑风暴]
各位大佬辛苦了，期待windows版本的编译

**Comment by BLumia** [comment]
期待windows版本的..啥？deepin 发行版的话，WSL 是有的，可以在微软商店获取。或者你可能想说 deepin IDE？

**Reply by whiteS18** [reply]
期待windows版本的IDE。
不好意思才发现他自动定位到这个地方了: (

**Reply by BLumia** [reply]
关联：

- https://github.com/linuxdeepin/developer-center/discussions/5430
- https://github.com/linuxdeepin/developer-center/discussions/8564
- https://github.com/linuxdeepin/developer-center/discussions/6630
- https://github.com/linuxdeepin/developer-center/discussions/5445
